.github/workflows/build_and_test.yml

```
@@ -271,3 +316,221 @@ jobs:
           path: |
             ~/${{ env.RESULT_PATH }}/results.html
             ~/${{ env.RESULT_PATH }}/results.xml
+
+  k8s_integration_tests:
+    name: k8s_integration_tests
+    runs-on: ubuntu-latest
+    needs: [helm_check, build]
+    strategy:
+      matrix:
+        # In K8S_VERSION, vX.XX.X is k8s version and after '-' X is CRI-O flag
+        K8S_VERSION: ["v1.15.0-0", "v1.16.0-0", "v1.17.0-0", "v1.18.0-0", "v1.19.0-0", "v1.19.0-1"]
```

Would just using `crio` will make this simpler?
```suggestion
        K8S_VERSION: ["v1.15.0", "v1.16.0", "v1.17.0", "v1.18.0", "v1.19.0", "crio"]
```
Then you can just set `WITH_CRIO` based on this matrix value?

---

Yes, let's use `crio` instead of specifying numeric flag in `K8S_VERSION`.
The above suggested changes are done can you please review them?